README.md

```diff
@@ -230,6 +230,10 @@ And another terminal with:
 script/ngrok
 ```
 
+You might want some features to be enabled, if it is the case then just run:
```

Comment:
@d12 What do you think of this part ? I might need some help with English with this one 😄

Reply:
How about `To enable optional features, run:`? 😄

Reply:
@d12 Oh. Yep. This one is more simple. Thanks and sorry 😅